### PR TITLE
Add Florence to team page

### PR DESCRIPTION
**Pull request by florenceyeun**
:eyeglasses: @spladug

### CODE REVIEW

File: reddit_about/data/team.json [hunk 1]
```diff
@@ -1138,6 +1138,21 @@
...
+            "description": "Hamster trainer.",
+            "favorite_subreddits": [
+                "/r/aww",
+                "/r/pottery"
+            ],
+            "name": "Florence Yeun",
+            "new": 201501,
+            "pyro": 0,
+            "role": "Software Engineer",
+            "role_details": "Software Hampster",
```
Comment (umbrae) [inline reddit_about/data/team.json]: Typo? Hamster?

**Comment by umbrae** [comment]
yay!

File: reddit_about/data/team.json (hunk 1 above)
Comment (florenceyeun) [inline reddit_about/data/team.json]: http://en.wikipedia.org/wiki/Hampster_Dance

**Comment by spladug** [comment]
:fish:

**Comment by florenceyeun** [comment]
:haircut:

**Comment by spladug** [comment]
:fish:

**Comment by florenceyeun** [comment]
0e353fb4461e5fbe776bbaa0937f033ec0751589

**Pull request closed without merging**